dockstore-integration-testing/src/test/java/io/dockstore/client/cli/NotificationIT.java

```diff
@@ -0,0 +1,133 @@
+        return notification;
+    }
+
+    @Test
+    public void testGetNotifications() {
+
+        // enter test notifications in the database
+        Notification expired = testNotification();
+        Notification current = anotherTestNotification();
+        curationApiAdmin.createNotification(expired);
+        curationApiAdmin.createNotification(current);
+
+        // get notifications and confirm that only the non-expired one is returned
+        List<Notification> activeNotifications = curationApiUser.getActiveNotifications();
+        assertEquals(activeNotifications.size(), 1);
+        assertEquals(activeNotifications.get(0).getMessage(), "ayy");
+    }
+
+    @Test
+    public void testCreateNewNotification() {
+
+        // set up a test notification
+        Notification notification = testNotification();
+
+        // try to create notification as an admin
+        Notification result = curationApiAdmin.createNotification(notification);
+        assertNotNull(result);  // createNotification should return a non-null notification
+        Assert.assertTrue(result.getId() > 0);  // if auto-increment is working, it should not overwrite row 0 of the database
+
+        // try to create notification as a regular user
+        boolean userCanCreate = true;
+        try {
+            curationApiUser.createNotification(notification);  // try to create as non-admin
+        } catch (ApiException e) {
+            userCanCreate = false;  // this should return a 400 error
+        }
+        assertFalse(userCanCreate);  // only admin/curators should be able to create notifications
+    }
+
+    @Test
+    public void testDeleteNotification() {
+
+        // create a test notification and add it to the database
+        Notification notification = curationApiAdmin.createNotification(testNotification());
+        long id = notification.getId();
+
+        // try to delete as non-admin
+        boolean userCanDelete = true;
+        try {
+            curationApiUser.deleteNotification(id);
+        } catch (ApiException e) {
+            userCanDelete = false;  // this should return a 400 error
+        }
+        assertFalse(userCanDelete);
+
+        // try to delete as an admin
+        curationApiAdmin.deleteNotification(id);
+        int count = testingPostgres.runSelectStatement(String.format("select count(id) from notification where id = '%s'", id), int.class);
+        assertEquals(0, count);  // confirm that there is no entry with this id in the database
+    }
+
+    @Test
+    public void testUpdateNotification() {
+
+        // create a test notification and add it to the database
+        Notification notification = curationApiAdmin.createNotification(testNotification());
+        long id = notification.getId();
+        System.out.println(id);
+
+        Notification update = anotherTestNotification();  // make another notification to use as the update
+        System.out.println("here");
```

Comment:
Probably don't want to leave this in as well as some other println's you have, although it's just a test, so not a huge deal.

Reply:
Alternatively, they can be switched to `LOG.debug` if there's a small chance they may be useful someday (probably not so much in this case, but for future FYI)